cli/src/commands/logs.ts

```diff
@@ -48,10 +55,27 @@ $ ${Command.cliBinName} logs --since 2m --follow`,
   public static strict = true;
 
   public async run() {
+    const { since, follow, limit } = this.parse(Logs).flags;
+    await this.authenticate();
 
-    const {flags: { since, follow, limit }} = this.parse(Logs);
-
-    this.log('Logs', { since, follow, limit });
+    const projectDir = await getProjectRootDir();
+    const projects = conf.get('projects') as Project[] | undefined || [];
```

Comment:
This might be simpler as
```suggestion
    const projects = (conf.get('projects') || []) as Project[];
```
It should certainly emit the same code.

Reply:
👍

Reply:
sorry. simpler, yes, but not good due to linting